std_msgs/Int32[] merge_group
